application/backend/src/main/resources/apps/typerefinery/components/content/columnticker/smallticker/script.js

```diff
@@ -0,0 +1,67 @@
+
+function getDataFromDataSourceSmallTicker(defaultJson, path, id,bgColor) {
+  const fetchAndUpdateView = async () => {
+   const jsonValue = defaultJson;
+    try {
+      const response = await fetch(path).then((res) => res.json());
+
+      !response.value ? smallTickerUpdateView(jsonValue, id,bgColor) : smallTickerUpdateView(response, id,bgColor);
+      console.log("data", response);
+    } catch (error) {
+        smallTickerUpdateView(jsonValue, id,bgColor);
+    }
+  };
+
+  fetchAndUpdateView();
+}
+
+function smallTickerUpdateView(jsonValue, id,bgColor) {
+  const smalltickerHtmlWithJsonValue = `
```

Comment:
Follow the camel case standars for all the variables.

Reply:
Done